tutorials/neuropixelsv2e-gui-tut.md

```diff
@@ -0,0 +1,190 @@
+
+In addition to the absolute zoom limits, the panel will automatically shift the probe to ensure it is always in view. This is handled each time the probe is zoomed or panned.
+
+To reset the view at any time, click on the `Reset Zoom` button to fully zoom out the panel.
+
+### Manually enabling electrodes
+
+Electrodes can be selected at any zoom level, but it is often preferable to zoom in to read the electrode indices. Consider maximizing the screen to see those numbers more easily.
+
+To select, as described [above](#maneuvering-along-the-probe), either click-and-drag the cursor over the desired electrodes, or select individual electrodes by clicking on them one-by-one. Once the electrodes to enable are selected, click on the `Enable Selected Electrodes` button in the right panel. At this point the selected electrodes should turn blue, indicating that they are now enabled. It is important to note that when electrodes are enabled, a number of previously enabled electrodes will be disabled due to channel constraints. For more information, read the [Channel constraints](#channel-constraints) section above.
+
+The short video below shows how to select, clear selection, enable selected electrodes, and translate using the scroll bar. Note that once electrodes are manually enabled, the `Channel Presets:` drop-down changes from `Shank0BankA` to `None`. Then, once the selected electrodes match the preset, it is automatically changed back to `Shank0BankA`.
+
+<div>
+  <video width="650" height="365" controls autoplay muted loop>
+    <source src="../images/neuropixelsv2e-gui-tut/probeconfiguration-enable-electrodes.webm">
+  </video>
+</div>
+
+### Loading and saving channel configurations
+
+When the GUI is first opened, and after a probe calibration file has been specified, the default [ProbeInterface](#probeinterface) configuration is loaded and drawn in the main panel. In this case, the default configuration is for a quad-shank `NeuropixelsV2e` probe, with the `Shank0BankA` channel preset selected. To load a new configuration, or load the default configuration, or save the current configuration, go to the File drop-down menu (see below) and choose the relevant option.
+
+<p align="center">
+  <img src="../images/neuropixelsv2e-gui-tut/probeconfiguration-filemenu.png">
+</p>
+
+#### Save ProbeInterface file
+
+To save a [ProbeInterface](#probeinterface) JSON file fully describing the probe, including which electrodes are currently enabled, go to the File drop-down menu, and select `Save Channel Configuration`. This will open a file dialog window to save the new JSON file. Choose a folder location and a name for the file, then hit `Save`. This will export the current channel configuration, and is a useful way to save any manually enabled electrodes as a backup, or to easily switch between different configurations between recordings.
```

Comment:
Nitpick. I would split the last sentence here into two at the first comma, but it's clear to me either way.